apps/src/templates/rubrics/StudentSelector.jsx

```diff
@@ -84,9 +84,16 @@ function StudentSelector({
           label: (
             <div className={style.studentDropdownOptionContainer}>
               <div className={style.studentDropdownOption}>
-                <BodyThreeText className={style.submitStatusText}>{`${
-                  student.name
-                } ${student.familyName || ''}`}</BodyThreeText>
+                <BodyThreeText className={style.submitStatusText}>
+                  {student.familyName
+                    ? student.familyName.length < 10
```

Comment:
A few questions about this solution...

1. Does this value (`10`) change depending on if a user is zoomed in or not?  Does the styling still break when zoomed in a bit?  Is that OK?
2. I am wondering about using the value of `10` here.  I feel like when Liam reviews my code he typically puts something like this in a constant at the top of the file.  How do you feel about that style choice here to avoid magic numbers?